.github/workflows/linux.yml

```diff
@@ -49,18 +49,19 @@ jobs:
         run:  sudo apt-get install -y tree $(./scripts/list-build-dependencies.sh -m apt -c gcc)
       - name: Log environment
         run:  ./scripts/log-env.sh
-      - name: Build
+      - name: Inject version string
         run: |
           set -x
           export VERSION=$(git describe --abbrev=4)
           sed -i "s/AC_INIT(dosbox,git)/AC_INIT(dosbox,$VERSION)/" configure.ac
+          echo ::set-env name=VERSION::$VERSION
+      - name: Build
+        run: |
+          set -x
           ./autogen.sh
           ./configure CXXFLAGS="-O3 -DNDEBUG"
```

Comment:
Add CFLAGS too, otherwise it defaults to `-g -O2`, so our single-headers built with optimizations too:
```
./src/libs/decoders/SDL_sound.c
./src/libs/decoders/flac.c
./src/libs/decoders/opus.c
./src/libs/decoders/vorbis.c
./src/libs/decoders/wav.c
./src/libs/decoders/xxhash.c
```

Reply:
Suggest adding `-pipe` to both, to eliminate intermediate files. 
In that github cache thread, the team manager admits their VM disks are extremely slow, so this keeps things in memory.